### DISCUSSION TITLE
Current version of Flameshot is useless

### DISCUSSION THREAD

**Discussion by ChuckNorrison** [flameshot-org/flameshot#3694] [General]
Iam a flameshot user for several years. In my workflow i click on the flameshot icon, choose screenshot and than i can grab an area and after that i can make some borders or text on my screenshot. This is gone and flameshot behaviour is exactly like the build in screenshot tool on my ubuntu 22 (wayland). 

Anyway, the current "Take Screenshot" and "Open Launcher" menu entries in the context menu are the same behaviour... this makes no sense at all. 

Iam very disappointed the last update broke my fancy flameshot :/

// I tested the mysterious "Share" button in the screen after taking a screenshot... here i get the edit mode for drawing some borders or place text or blurr something. So it feels it needs more steps to get there and a screenshot is always saved to disk and maybe uploaded somewhere. I dont like to throw images into the cloud silently, so i set the automatically generated imgur id to xxx. This feels a bit creepy.

In older versions i just pressed CTRL + S if i want to save screenshot anywhere. Mostly i use CTRL + C to get it into the clipboard to paste it in a chat. 

Conclusion:
Would be great to create a context menu entry to get straight into the edit capture mode

**Comment by ChuckNorrison** [comment]
I found this to the topic, which suggests the behaviour is related to gnome 41 and later. So the behaviour changed with updating my systems gnome. 

https://flameshot.org/docs/guide/wayland-help/#i-am-asked-to-share-my-screen-every-time

This gets adressed here
https://github.com/flatpak/xdg-desktop-portal/issues/649

**Reply by mmahmoudian** [reply]
Good that you find the documentation. Of course if you are disappointed, switch from Gnome to something less opinionated and be in control of your computer.

Also the portal link you shared only worka for Flatpak. It doesn't fix the Gnome issue completely (e.g if you install via apt, snap, AppImage,...)
